store/tikv/coprocessor.go

```diff
@@ -1294,6 +1293,7 @@ type rateLimitAction struct {
 		// isTokenDestroyed indicates whether there is one token has been isTokenDestroyed after Action been triggered
 		isTokenDestroyed bool
 		once             sync.Once
+		triggerCount     uint
```

Comment:
1. add a comment for this attribute
2. triggerCount --> triggerCountForTest

Reply:
updated.